pm/sendermonitor.go

```diff
@@ -79,11 +78,13 @@ type senderMonitor struct {
 	// each of currently active remote senders
 	redeemable chan *SignedTicket
 
+	ticketStore func() TicketStore
```

Comment:
SenderMonitor only needs to pass this dependency to newly created ticketqueues so rather than storing it as state I felt it would be okay to wrap it in a function